scripts/activate-venv.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# run with 'source scripts/activate-venv.sh'
+set -e
+
+ROOT="$(cd "$(dirname "$0")"/.. && pwd)"
+ENV_NAME=".py_venv"
+
+# if venv does not exist, send error message and exit
+if [ ! -d "${ROOT}/${ENV_NAME}" ]; then
+  echo "Virtual environment '${ENV_NAME}' not found."
+  if [[ $OSTYPE == "linux-gnu"* ]]; then
+    echo "run 'sudo ./scripts/install-build-tools.sh' to create it."
+  elif [[ $OSTYPE == "darwin"* ]]; then
+    echo "run './scripts/install-build-tools.sh' to create it."
+  fi
+  exit 1
```

Comment:
I only just realized this script is not intended to ever be run directly, only to be `source`d. It should *never* call `exit` in that case. This resulted in closing the shell I was using to test.

Reply:
ACK - good point, fixed

Reply:
I'll test again tomorrow; I suspect that misunderstanding/confusion is what tripped up my test.